chart/kubeapps/values.yaml

```diff
@@ -1658,6 +1660,12 @@ kubeappsapis:
   ## @param kubeappsapis.replicaCount Number of frontend replicas to deploy
   ##
   replicaCount: 2
+  ## @param kubeappsapis.qps KubeappsAPIs Kubernetes API client QPS limit
+  ##
+  qps: "10.0"
```

Comment:
Out of interest, I notice that for kubeops, the chart defaults are "" (ie. not set by the chart) with the kubeops binary setting defaults of 10 and 15 when the args aren't set, but here for kubeapps apis you're explicitly setting the defaults here?

Reply:
Exactly,  that's precisely the intention: in kubeops, we were modifying the default rate limit values directly in the code, so users wouldn't notice the actual value being used.

If we set the values in the values.yaml, it will appear directly in the readme, so users will immediately know which is the default value without digging into the code.


> Hmm, why default to 10.0 then?

In this PR I'm just using the same values as we were defining in Kubeops. The proper value for the kubeappsapis service to work decently is yet to be analyzed and defined (100 was just a random feasible value, but we have to give it a thought)